main/modes/music/jukebox/jukebox.c

```diff
@@ -69,6 +83,15 @@ typedef struct
     const uint8_t numSongs;
 } jukeboxCategory_t;
 
+typedef struct
+{
+    wsg_t* graphic;
+    int16_t x;
+    double y;
```

Comment:
Usually I'd nitpick about `double` performance, but this is just a jukebox, not a game.